<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndWindow</class>
 <widget class="QMainWindow" name="EndWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="back_to_menu">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>940</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 返回主菜单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>1030</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>811</width>
      <height>1231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Text1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>最终成绩：</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>701</width>
      <height>801</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>游戏结束……</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>back_to_menu</zorder>
   <zorder>exit</zorder>
   <zorder>Text1</zorder>
   <zorder>scoreLabel</zorder>
   <zorder>Text2</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
